src/Aks/Aks.sln

```diff
@@ -1,7 +1,6 @@
-
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.27703.2035
+# Visual Studio Version 16
```

Comment:
Shall we change this file?

Reply:
Good question. As we stated VS 16/2019 is [prerequisite](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/azure-powershell-developer-guide.md#prerequisites) for our project, I would suggest to upgrade all solution files to VS 16.